plugins/deep-link/src/config.rs

```diff
@@ -10,10 +10,20 @@ use serde::{Deserialize, Deserializer};
 
 #[derive(Deserialize)]
 pub struct AssociatedDomain {
+    #[serde(default)]
```

Comment:
shouldn't we default this to at least `https` (and maybe `http` too at least on debug)?